src/test/java/org/apache/xml/security/test/dom/interop/BaltimoreTest.java

```diff
@@ -277,37 +277,6 @@ public void test_fifteen_external_dsa() throws Exception {
         assertTrue(verify, filename);
     }
 
-    /**
```

Comment:
I think it can be useful to modify and keep this as a negative test, that is make sure it throws an exception when Xalan is not available to process the here() function.

Reply:
Yep fixed now